types/downscale/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for downscale 1.0
+// Project: https://github.com/ytiurin/downscale
```

Comment:
This seems like an umd module. You're probably missing the `export as namespace downscale` to be used correctly.

Reply:
yep, you're right!